docs/design/y-flow/y-flow-design.md

```diff
@@ -0,0 +1,72 @@
+# Y-flows
+
+## Concepts / objects model
+
+New kilda entity - Y-flow. Defined by 3 network endpoint. One "shared" endpoint defined by
+switch+port+outer-vlan+inner-vlan-A+inner-vlan-B. Two other endpoints defined by switch+port+vlan (can match or can
```

Comment:
Let's discuss this offline, afair this shared endpoint can't be fully matched.

Reply:
Discussed.